chapters/pt/_toctree.yml

```diff
@@ -2,6 +2,31 @@
   sections:
   - local: chapter0/1
     title: Introdução
+
+- title: 1. Modelos Transformers
+  sections:
+  - local: chapter1/1
+    title: Introdução
+  - local: chapter1/2
```

Comment:
We should only include in the table of contents the sections that have a corresponding .mdx file. So in this case, everything from this line and below can be removed :)